src/Azure/Orleans.Streaming.AzureStorage/Providers/Streams/AzureQueue/IAzureQueueDataAdapter.cs

```diff
@@ -73,7 +73,7 @@ public AzureQueueDataAdapterV2(SerializationManager serializationManager)
         /// <summary>
         /// Creates a cloud queue message from stream event data.
         /// </summary>
-        public CloudQueueMessage ToQueueMessage<T>(Guid streamGuid, string streamNamespace, IEnumerable<T> events, Dictionary<string, object> requestContext)
+        public CloudQueueMessage ToQueueMessage<T>(Guid streamGuid, string streamNamespace, IEnumerable<T> events, StreamSequenceToken token, Dictionary<string, object> requestContext)
```

Comment:
Is this AzureQueueDataAdaptor related change to this pr?

Reply:
Yes, unfortunately.  The interface surface changed, thus so need the implementations.